sample/Kmd.Logic.Cpr.Client.Sample/Program.cs

```diff
@@ -50,6 +50,12 @@ private static void InitLogger()
 
         private static async Task Run(AppConfiguration configuration)
         {
+            var cprPersonId = Guid.Parse("5e3d9df6-d082-467a-88bd-dca56edc7328");
```

Comment:
No need for this. Use the Id off the citizen object

Reply:
removed and making use of citizen object